src/include/packager/repositories_include.rb

```diff
@@ -204,8 +210,8 @@ def createSourceImpl(url, plaindir, download, preffered_name, force_alias)
           end
           alias_name = ""
           if force_alias == ""
-            # replace " " -> "_" (avoid spaces in .repo file name)
-            alias_name = Builtins.mergestring(Builtins.splitstring(name, " "), "_")
+            # replace " " -> "_" (avoid spaces in .repo file name) and remove shell unfriendly '/()'
```

Comment:
NP: Update the comment as well, `remove shell unfriendly characters` sounds good enough.